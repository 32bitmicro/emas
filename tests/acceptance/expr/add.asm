.word 1 + 1
.word 1.0 + 1.0
.word -1 + -1
.word -1.0 + -1.0
.word 65534 + 1
.word -32769 + 1
.word 32767+1
