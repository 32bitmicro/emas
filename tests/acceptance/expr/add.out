@ 0x0000 : 0x0002  /  000 000 0 000 000 010  /  2
@ 0x0001 : 0x0002  /  000 000 0 000 000 010  /  2
@ 0x0002 : 0xfffe  /  111 111 1 111 111 110  /  -2
@ 0x0003 : 0xfffe  /  111 111 1 111 111 110  /  -2
@ 0x0004 : 0xffff  /  111 111 1 111 111 111  /  65535
@ 0x0005 : 0x8000  /  100 000 0 000 000 000  /  -32768
@ 0x0006 : 0x8000  /  100 000 0 000 000 000  /  32768
